src/doc_builder/utils.py

```diff
@@ -74,3 +76,95 @@ def get_doc_config():
     Returns the `doc_config` if it has been loaded.
     """
     return doc_config
+
+
+def convert_parametype_numpydoc_to_groupsdoc(paramtype):
+    """
+    Converts parameter type str that is written in numpystyle to groupsstyle (e.g. transformers docs).
+
+    Args:
+    paramtype (`str`): The name of the object to document.
+
+    Example:
+    ```py
+    >>> s = "bool, optional, default True"
+    >>> convert_numpydoc_to_rstdoc(s)
+    '`bool`, `optional`, defaults to `True`'
```

Comment:
This example needs to be updated.

Reply:
handled in https://github.com/huggingface/doc-builder/pull/89/commits/232277e582f84dd6f9d6b710a7ad118ef2636e46